homeassistant/components/scene/__init__.py

```diff
@@ -32,15 +32,13 @@ def _hass_domain_validator(config: dict[str, Any]) -> dict[str, Any]:
 
 def _platform_validator(config: dict[str, Any]) -> dict[str, Any]:
     """Validate it is a valid  platform."""
+    platform_name = config[CONF_PLATFORM]
     try:
-        platform = importlib.import_module(f".{config[CONF_PLATFORM]}", __name__)
+        platform = importlib.import_module(
+            f"homeassistant.components.{platform_name}.scene"
```

Comment:
Are there any platforms left besides `homeassistant`?

Reply:
I count 20 platforms including homeassistant.

Reply:
Sorry, I should clarify. Do any of them setup from PLATFORM_SCHEMA?

Reply:
```
homeassistant/scene.py:async def async_setup_platform(
lifx_cloud/scene.py:async def async_setup_platform(
```

I guess lifx_cloud is the only one besides homeassistant

Reply:
If someone loads that one it will do blocking I/O to import the platform during `async_process_component_config`.  I'm not sure its worth fixing though since analytics says there is 1 user https://www.home-assistant.io/integrations/lifx_cloud/

Reply:
Maybe we could reorder the config process to import the platform first so its already loaded in time

Reply:
https://github.com/home-assistant/core/tree/config_load_platform_before_validation

That seems to work ok and would broadly solve the issue

Will need to do more testing with it after some sleep

Reply:
Maybe not worth the change risk there since I think this issue affects only a handful of instances 

Something to sleep on

Reply:
After some sleep, I don't think changing config can be justified for a single integration with negligible usage which will go away as soon as its converted to a config flow (or removed because the cloud service shuts down). I think this change is going to be enough to resolve the problem.